test/lint/streams.rb

```diff
@@ -633,6 +634,72 @@ def test_xclaim_with_invalid_arguments
       assert_raises(Redis::CommandError) { redis.xclaim('', '', '', '', '') }
     end
 
+    def test_xautoclaim
+      omit_version(MIN_REDIS_VERSION_XAUTOCLAIM)
+
+      redis.xadd('s1', { f: 'v1' }, id: '0-1')
+      redis.xgroup(:create, 's1', 'g1', '$')
+      redis.xadd('s1', { f: 'v2' }, id: '0-2')
+      redis.xadd('s1', { f: 'v3' }, id: '0-3')
+      redis.xreadgroup('g1', 'c1', 's1', '>')
+      sleep 0.01
```

Comment:
What's the sleep for?

Reply:
`XPENDING` and `XAUTOCLAIM` both return entries in the consumer group that have been idle for `n` ms. We need them to be idle for a few ms in order to get anything in the test.

Reply:
👍